tcl/target/rp2040.cfg

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 # RP2040 is a microcontroller with dual Cortex-M0+ core.
-# https://www.raspberrypi.com/documentation/microcontrollers/rp2040.html
```

Comment:
The RP2040 "documentation URL" (as opposed to the "product URL") would seem to be https://www.raspberrypi.com/documentation/microcontrollers/silicon.html#rp2040